ui/types/vault/models/pki/issuer.d.ts

```diff
@@ -3,9 +3,10 @@ import { FormField, FormFieldGroups, ModelValidations } from 'vault/app-types';
 export default class PkiIssuerModel extends PkiCertificateBaseModel {
   useOpenAPI(): boolean;
   issuerId: string;
+  issuerName: string;
+  issuerRef: string;
```

Comment:
Since this is a getter now I think it should be `issuerRef(): string`. Not 100% sure but noticed that `useOpenAPI` is declared that way.

Reply:
Good point! Thank you